MAUI/DataForm/Editing.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: post
+title: Data handling in .NET MAUI DataForm control | Syncfusion
+description: Learn here all about the commit feature in Syncfusion .NET MAUI DataForm (SfDataForm) control and more.
```

Comment:
title and description - can you improve based on the discussion @Jeyasri-Murugan ?